modules/ROOT/pages/introduction/intro-connector-configuration-overview.adoc

```diff
@@ -0,0 +1,148 @@
+
+. In the connector global element *Choose Global Type* screen, search for the element called *Configuration properties*.
+. Select *Configuration properties*.
+. Click *OK*.
+. Click *…​* to the right of the *File* field and navigate to your YAML or properties file.
+
+To reference a file that stores property values when configuring a connector using XML:
+
+. Include a `<configuration-properties>` element inside the `<mule>` element.
+. Set the `file` parameter with your properties file name. For example:
+
+[source,xml,linenums]
+----
+<mule>
+  <configuration-properties
+    file="myConfiguration.yaml"
+    doc:name="Configuration properties"
+    />`
+  ...
+</mule>
+----
+
+For more information about using property placeholders, see xref:mule-runtime::create-and-configure-your-properties-file.adoc[Configuring Property Placeholders].
+
+[[test-connectivity]]
+=== Test the Connectivity of the Connector
+
+Both Studio and Flow Designer enable you to test the connection between a connector and its data source by clicking a button.
+
+In Studio, after you configure the required connection values in the *Global Element Properties* screen, click *Test Connection*:
+
+.Click the *Test Connection* button in Studio to test the connection
+image::introduction/intro-config-overview-studio-test.png[Test the connection in Studio]
+
+In Flow Designer, after you configure the required connection values in the *Configure Connection* screen, click *Test*:
+
+.Click the *Test* button in Flow Designer to test the connection
+image::introduction/intro-config-overview-fd-test.png[Test the connection in Flow Designer]
+
+[[reconnection-strategy]]
+=== Configure a Reconnection Strategy for the Connector
+
+When a Mule app starts, a connectivity test runs on connectors that must connect to an external server. If the test fails, the default behavior is to log a warning message and continue with the deployment of the app, instead of blocking the deployment.
+
+You can modify this behavior by setting the number of reconnection attempts to try after an initial failure and forcing the deployment to fail when all connection attempts fail.
+
+To configure a reconnection strategy for a connector using Studio, use the *Advanced* tab in the global element configuration *General* tab. The following example configures a reconnection strategy that:
+
+* Forces the deployment to fail if the connection fails and any configured reconnection attempts are exhausted
+* Limits the number of reconnection attempts to 5 at a frequency of every 4000 milliseconds (ms)
+
+.Use the fields in the Reconnection section of the global element to specify a reconnection strategy
+image::introduction/intro-config-overview-reconnection-strategy-studio.png[Reconnection strategy example using Studio]
+
+To configure a reconnection strategy for a connector using Flow Designer, use the *Advanced* tab in the global element configuration. The following example configures a reconnection strategy that:
```

Comment:
"...in the global element configuration..." page?

Reply:
Correct as is.

Reply:
OK